test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -270,7 +274,7 @@ private static string ToSafePathString(string value)
 
                             foreach (var (path, content) in commit.Files)
                             {
-                                var blob = repo.ObjectDatabase.CreateBlob(new MemoryStream(Encoding.UTF8.GetBytes(content ?? "")));
+                                var blob = repo.ObjectDatabase.CreateBlob(new MemoryStream(Encoding.UTF8.GetBytes(content?.Replace("\r", "") ?? "")));
```

Comment:
>s(content?.Replace("\r", "") [](start = 112, length = 28)

why '\r' needs to be removed?

Reply:
no need actually, will remove

---
In reply to: [231405628](https://github.com/dotnet/docfx/pull/3585#discussion_r231405628) [](ancestors = 231405628)